### ISSUE TITLE
Seems Bracket Pair Colorizer 2 still doesn't work

### ISSUE THREAD

**Issue by KevinZonda** [cdr/code-server#2579]
- Web Browser: Chrome 87
- Local OS: Windows 10 1909
- Remote OS: Debian 10
- Remote Architecture: x64
- `code-server --version`: 3.8.0 c4610f7829701aadb045d450013b84491c30580d

I tried linking `node_modules.asar` to `node_modules` as well(#2197), but my Bracket Pair Colorizer 2 still doesn't work

![image](https://user-images.githubusercontent.com/33132228/104535791-bc15ac80-560e-11eb-975a-c0231a59999f.png)

(PS: Whatever I link the `node_modules.asar` to `node_modules` , the situation seems always be the same, so I only post one. I also tried to open code-server's log to trace, but nothing special happened.)

**Comment by code-asher** [comment]
I tried to replicate on 3.8.0 but no luck yet. Could you post the
browser console log and code-server's output?

**Comment by KevinZonda** [comment]
![image](https://user-images.githubusercontent.com/33132228/104645499-064a6c80-56a7-11eb-83a0-56b4b201b5be.png)

I set the Chrome log level to Warnings and Errors, and such information is printed.
The first line is from my anti-virus software.

**Comment by KevinZonda** [comment]
I dont know what is code-server's output... Is that run like `code-server --log debug` and get the output, or what?

**Comment by code-asher** [comment]
Yes exactly! But I don't think it'l actually be necessary as it turns
out I can actually replicate on Rust files. I was using a Typescript
file previously.

**Comment by KevinZonda** [comment]
Exactly, I tried JavaScript files, seems works fine, only Rust doesn't work
